code/model/UserDefinedForm.php

```diff
@@ -60,6 +60,18 @@ class UserDefinedForm extends Page {
 	);
 
 	/**
+	 * @var Boolean
+	 */
+	private static $recipients_warning_enabled = false;
+
+	/**
+	 * @return Boolean
+	 */
+	public static function set_recipients_warning_enabled($bool = true) {
```

Comment:
You need to use the config api to change the config value; In fact it would be better to just not have the setter, and document the fact that it's config driven. :D We don't really use custom setters with config these days (they are just needless overhead when yml config works just as well).